src/unitTest/PageParserTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+include_once '../parserPages/PageParser.php';
+include_once '../dataBase/XmlFeedModel.php';
+
+use PHPUnit\Framework\TestCase;
+
+class PageParserTest extends TestCase
+{
+    public function testPageParser()
+    {
+        $objectPage = new PageParser();
+        $sourceLink = 'https://usn.ubuntu.com/usn/rss.xml';
+        $this->assertNotEmpty($sourceLink, 'This link empty!');
+        $haystack = $objectPage->getValuesInItem($sourceLink);
```

Comment:
It would be good to add a test case where you pass invalid url and check the result. Another case is when the url does not contain valid xml object.
Please add 3-4 more test cases here. They should also be separated by different functions.